index.js

```diff
@@ -14,11 +14,10 @@
  * Module dependencies.
  */
 
-var os = require('os');
+var osName = require('os-name');
 
-var USER_AGENT = 'node/' + process.version
-  + ' (' + process.platform + ' ' + os.release()
-  + '; ' + process.arch + ')';
+var USER_AGENT = 'Node.js/' + process.version.slice(1)
```

Comment:
Ok, that new stuff was added to io.js v3.0.0, but what good is it to us when io.js *is* officially Node.js now?